params/bootnodes.go

```diff
@@ -21,26 +21,26 @@ import "github.com/dominant-strategies/go-quai/common"
 // ColosseumBootnodes are the enode URLs of the P2P bootstrap nodes running on
 // the Colosseum test network.
 var ColosseumBootnodes = []string{
-	"enode://1edee38aebefea9150213cdf4306a64499e3bca9058971c443d31a4997b9a096e7a26059472529a6ccbca0a04c4f6fdbf0c880ee97545a12c0f8b656f7d4e7da@35.192.126.165",
-	"enode://0f7b74f957d68eb85bbbf8ffe7d0477575406beb647f2353de08ca291a03db1507a70a41f3b4623c3049512342781c8fa9a7f7654cbcc8603e51190430b839a6@34.170.190.144",
-	"enode://66c87db18ab3e6322a149ac4b7b1bde3a31f1e505945a358e4c1ebae9dc8a40402ccd96dfed35438448fbf79a7b8ee093581a05c40ab115adaebe81a6946993b@104.197.16.107",
+	"enode://aa76ab25bd9657efc8371103649ad9fc099388f1b303b18ab9b5fbdd43c8bae1e039357eb6a5a3aa86679f58ed0231c88bd207393448a7226d63fdc4651698cb@35.202.3.128",  // us-central1-a
+	"enode://7a80da93d6ba5299fccf4dc195c0a9efaee5ef3ef9ae00c7ae4db72f10cd26750b7acaf956a381f3101860b58c931b56a4524051440382f1be5e33dc247a9b52@35.184.84.143", // us-central1-a
+	"enode://1ba881701345af5491ec9ced31d2c325bdd296219015856109e8405bfa51665c0dc91b85a93541ba2ae6294d0dee4ef9c8cab7364594234447b30237631e90c6@35.224.101.46", // us-central1-a
 }
 
 // GardenBootnodes are the enode URLs of the P2P bootstrap nodes running on the
 // Garden test network.
 var GardenBootnodes = []string{
-	"enode://75c6cbbb08e45281606fd963c1fbbad763227dc370a9bf36185c7d9e9a66b7f7eebb17572d6932fb108ec37f333e8703656af79a98e651040131174a50124d09@34.64.47.60",   // Asia
-	"enode://273a1c529bdf064acbae6ab29c5c533f8caea2dce0ea4b0d5b7b6da58cc5b0de5924cd30bced5aba231a0aa9e5063d3baddc6d77e924d3999640a9c1304c5ef1@34.175.50.212", // Europe
-	"enode://e5c5470889825d2d32d6f55ee74d50d7d0040407f1339123572e675c818cc64dcbfbb4e202bc63bdff5e80e6c9ebba81370588d9737d127fbb663614ab7ece62@34.95.212.143", // SouthAmerica
-	"enode://b270dad3ea88fd8276d2eb20403c0f34d3eb1765f7dc124c3c41ed862f6adf78789b41ee7a7f0117f9e82a5c0a8ae3d9dae3d7c9200ea62cb82af9af08f36cfb@34.70.251.243", // Central USA
+	"enode://f353567741755a4b35afb4641618b34c7e3c1666b49c9e519e8336c124ba8c2c556f00e9edcd87599994e9674ebdea3503a5ff1948a257c1540fe9f8c0fedf62@34.22.72.242",  // Asia
```

Comment:
Can we update the region comments to be just as specific as for Orchard & garden?

Reply:
Resolved as requested